test/integration/disruption/disruption_test.go

```diff
@@ -498,9 +502,15 @@ func waitToObservePods(t *testing.T, podInformer cache.SharedIndexInformer, podN
 }
 
 func TestPatchCompatibility(t *testing.T) {
-	s, _, _, clientSet, _, _ := setup(t)
+	s, pdbc, _, clientSet, _, _ := setup(t)
```

Comment:
don't you need to start the informers too?

Reply:
we don't - just creating informers doesn't start any goroutines - so we don't need to touch them